src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin.cxx

```diff
@@ -107,14 +190,47 @@ PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin<Tar
 set_up_before_sensitivity(shared_ptr <TargetT > const& target_sptr) 
 { 
 #ifdef STIR_MPI
-        //broadcast objective_function (100=PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin)
-        distributed::send_int_value(100, -1);
+    //broadcast objective_function (100=PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin)
+    distributed::send_int_value(100, -1);
 #endif
-        
- 
-  // set projector to be used for the calculations    
-  this->PM_sptr->set_up(this->proj_data_info_cyl_uncompressed_ptr->create_shared_clone(),target_sptr); 
-  return Succeeded::yes;
+
+
+    // set projector to be used for the calculations
+    this->PM_sptr->set_up(proj_data_info_cyl_sptr->create_shared_clone(),target_sptr);
+
+    shared_ptr<ForwardProjectorByBin> forward_projector_ptr(new ForwardProjectorByBinUsingProjMatrixByBin(this->PM_sptr));
```

Comment:
Given that you reverted some stuff, we have 2 choices here.

- the `compute_gradient*` doesn't use it. Sensitivity only uses backprojection. just create the backprojector.
- still create a projector pair, but use `ProjectorPairUsingProjMatrixByBin`

I guess the 2nd option is better for future compatibility with replacing the sensitivity stuff with `distributable_computation*`

Reply:
Done the second option